apps/web/src/server/views/appeals/appeal/appeal-details.njk

```diff
@@ -2,6 +2,7 @@
 
 {%- from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner -%}
 {%- from "govuk/components/summary-list/macro.njk" import govukSummaryList -%}
+{% from "govuk/components/table/macro.njk" import govukTable %}
```

Comment:
Super pedantic point, but just for consistency could you use `{%-` and `-%}` here? The difference is whitespace control, although I don't think it matters in this case, I generally prefer to use this everywhere: https://mozilla.github.io/nunjucks/templating.html#whitespace-control

Reply:
done